docs/integration/RuoYi.md

```diff
@@ -0,0 +1,201 @@
+---
+title: RuoYi
+description: Using Casdoor in RuoYi-Cloud
+keywords: [RuoYi]
+---
+
+Casdoor can connect to RuoYi-cloud simplely
+
+## Step1. Deploy Casdoor
+
+Firstly, the Casdoor should be deployed. 
+
+You can refer to the Casdoor official documentation for the [Server Installation](/docs/basic/server-installation).
+
+After a successful deployment, you need to ensure:
+
+- The Casdoor server is successfully running on **http://localhost:8000**.
+- Open your favorite browser and visit **http://localhost:7001**, you will see the login page of Casdoor.
+- Input `admin` and `123` to test login functionality is working fine.
+
+Then you can quickly implement a casdoor based login page in your own app with the following steps.
+
+
+## Step2. Configure Casdoor 
+Configure casdoor can refer to [casdoor](https://door.casdoor.com/login)(Configure casdoor's browser better not use one browser with your develop browser).
+
+You also should configure organization, role, permissions, modle, application, Synchronizer, you also can refer to [casdoor](https://door.casdoor.com/login).
```

Comment:
In this section, it is better to show an example for user to follow and config.
And refer to specific docunment, like [application](https://casdoor.org/docs/application/config), [Synchronizer](https://casdoor.org/docs/syncer/overview).

Reply:
@jakiuncle plz improve this section.